addons/hr/models/res_users.py

```diff
@@ -295,3 +295,18 @@ def action_create_employee(self):
             company_id=self.env.company.id,
             **self.env['hr.employee']._sync_user(self)
         ))
+
+    def action_open_employees(self):
+        self.ensure_one()
+        employees = self.env["hr.employee"].search([('user_id', '=', self.id)])
```

Comment:
```suggestion
        employees = self.employee_ids
```
or directly
`if len(self.employee_ids) > 1:`